lib/index.js

```diff
@@ -2,6 +2,21 @@
 
 const _ = require('lodash');
 
+const isTruthy = (model, field) => {
+  field = typeof field === 'object' ? field.field : field;
+  const value = model.get(field);
+  return value !== '' && value !== null && value !== undefined;
```

Comment:
all those values are falsey, you could reduce that down to `return Boolean(value);`